### ISSUE TITLE
Can't access to mounted smartphone

### ISSUE THREAD

**Issue by sato96** [GSConnect/gnome-shell-extension-gsconnect#1678]
### Describe the bug

Using old issues i can mount my realme 11 pro + but i can't open: 
when from nautilus i try to open it i get this error (translated from italian):
_"You don't have enough permission to see content from <</  on 192.168.1.20>>"_
If i try to launch a nautilus istance from terminal with root permission i can't find my smartphone.
Here is my Log:

```
GSConnect: 50 (system)
GJS:       17202
Session:   wayland
OS:        Ubuntu 22.04.3 LTS
--------------------------------------------------------------------------------
set 17 01:18:01 dbus-daemon[1585]: [session uid=1000 pid=1585] Activating service name='org.gnome.gedit' requested by ':1.107' (uid=1000 pid=3223 comm="gjs /usr/share/gnome-shell/extensions/gsconnect@an" label="unconfined")
set 17 01:18:01 dbus-daemon[1585]: [session uid=1000 pid=1585] Successfully activated service 'org.gnome.gedit'
```

### Steps to reproduce

1. Mount your smartphone
2. Try to open it

### Expected behavior

My smartphone filesystem

### GSConnect version

50

### Installed from

OS package manager

### GNOME Shell version

42.9

### Linux distribution/release

Ubuntu 22.04.3 LTS

### Paired device(s)

realme 11 PRO +

### KDE Connect app version

1.27.1

### Support log

```plain text
GSConnect: 50 (system)
GJS:       17202
Session:   wayland
OS:        Ubuntu 22.04.3 LTS
--------------------------------------------------------------------------------
set 17 01:18:01 dbus-daemon[1585]: [session uid=1000 pid=1585] Activating service name='org.gnome.gedit' requested by ':1.107' (uid=1000 pid=3223 comm="gjs /usr/share/gnome-shell/extensions/gsconnect@an" label="unconfined")
set 17 01:18:01 dbus-daemon[1585]: [session uid=1000 pid=1585] Successfully activated service 'org.gnome.gedit'
```
```


### Screenshots


![Schermata del 2023-09-17 01-26-12](https://github.com/GSConnect/gnome-shell-extension-gsconnect/assets/56189477/7f658a3b-8f7f-4a56-9ae8-3c47c06c5c29)



### Notes

_No response_

**Comment by dkowis** [comment]
I think I'm having the same problem:

![image](https://github.com/GSConnect/gnome-shell-extension-gsconnect/assets/28048/e5619f07-ba4b-420f-9b43-7d3d9175f406)


I have it mounted finally after doing the fix for #1203, but unfortunately, I can't actually do anything with it :(

**Comment by ivanzgroot** [comment]
Same issue here on Debian 12, GSConnect 54 and Gnome 43.6. 
I Installed the extension, paired the device, fixed the rsa issue #1203 and tried to mount the FS of the device (KDE Connect 1.27.1 on Pixel 7a). 

![Screenshot from 2023-09-25 23-21-46](https://github.com/GSConnect/gnome-shell-extension-gsconnect/assets/101365119/e1f37611-bdea-4b9c-bf33-54c9ccd43b43)

**Comment by sato96** [comment]
I found another issue like mine: #1610 so i can close this one
